Granicus.MediaManager.UserSDK/MediaManager.cs

```diff
@@ -117,7 +117,7 @@ private string m_SafeServerURL(string Server)
             if (!Server.StartsWith("https://") && !Server.StartsWith("http://"))
             {
                 Server = "https://" + Server;
-            } else if(Server.StartsWith("http://")) {
+            } else if(Server.StartsWith("http://") && !Server.Contains("mm.lvh.me")) {
```

Comment:
Being really nitpicky, but can we move "mm.lvh.me" into a descriptive constant, like DevURL or something?